Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -251,5 +284,26 @@ void OnDestroy()
             InteractionManager.SourceUpdated -= InteractionManager_SourceUpdated;
             InteractionManager.SourceLost -= InteractionManager_SourceLost;
         }
+
+        public void OnSelect()
```

Comment:
That's the mystery of SendMessage (exactly why I've requested we remove these).
They get called by SendMessage("OnSelect").
If we move this into a new class we'd have to make sure that class was also included on the same GameObject as our GestureManager.cs and GazeManager.cs See [Unity Documentation](https://docs.unity3d.com/ScriptReference/GameObject.SendMessage.html)

> Calls the method named methodName on every MonoBehaviour in this game object.